vendor/bat-native-ledger/src/bat/ledger/internal/static_values.h

```diff
@@ -25,11 +25,11 @@
 "https://balance.rewards.brave.software"
 
 #define PUBLISHER_STAGING_SERVER     \
-"https://publishers-staging-distro.basicattentiontoken.org"
+"https://pcdn.brave.software"
```

Comment:
Staging should be `pcdn.bravesoftware.com`.

`pcdn.brave.software` is dev, as correctly defined below.

Reply:
@fmarier I'm getting an error fetching from `pcdn.bravesoftware.com`. I believe devops only setup production and dev

Reply:
Should be fixed now: https://bravesoftware.slack.com/archives/C011BLDBB63/p1591895026073400

Reply:
Fixed